docs/source/ingest/destination_connectors/sql.rst

```diff
@@ -0,0 +1,99 @@
+PostgreSQL
+===========
+NOTE: At the moment, the connector only supports PostgreSQL.
+
+Batch process all your records using ``unstructured-ingest`` to store structured outputs locally on your filesystem and upload those local files to a PostgreSQL schema.
+
+First you'll need to install the sql dependencies as shown here.
+
+.. code:: shell
+
+  pip install "unstructured[sql]"
+
+Run Locally
+-----------
+The upstream connector can be any of the ones supported, but for convenience here, showing a sample command using the
+upstream s3 connector.
+
+.. tabs::
+
+   .. tab:: Shell - with parameters
+    
+    You can use the connectors with the same parameters used in `sqlalchemy.engine.URL.create <https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.engine.URL.create>`_
+
+      .. code:: shell
+
+        unstructured-ingest \
+            s3 \
```

Comment:
Can we update this to use the local source connector to match the destination ingest tests?

Reply:
Doesn't seem like this comment has been fulfilled.